### DISCUSSION TITLE
How Blender uses ZLUDA?

### DISCUSSION THREAD

**Discussion by Qqqqq77** [vosen/ZLUDA#222] [Q&A]
I would like to know how to render with ZLUDA GPU on Blender, my GPU is 7900XTX, can you explain the steps?I've been looking online for a long time and haven't found a way.This will help me a lot.

**Comment by st0pman** [comment]
https://youtu.be/p7fIUXc0KJg?si=jFEiT82OxTbWU2wI

**Comment by st0pman** [comment]
He is Russian, but if it doesn't matter, then watch the video it helped me, and Blender 4.1 also doesn't work with stock Zluda

**Comment by Qqqqq77** [comment]
Thank you！发自我的 iPhone在 2024年7月19日，02:40，st0pman ***@***.***> 写道：﻿
https://youtu.be/p7fIUXc0KJg?si=jFEiT82OxTbWU2wI
